src/role_harvester.js

```diff
@@ -102,6 +102,10 @@ roles.harvester.preMove = function(creep, directions) {
   if (directions && creep.memory.routing.reverse) {
     directions.direction = directions.backwardDirection;
   }
+  if (directions.backwardDirection && reverse && roles.scout.isStuck(creep)) {
```

Comment:
The `isStuck` method should be on the creep, not on the scout.

Reply:
👍

Reply:
👍 yeah, was firstly made for period when harvesters are multiples in rooms and stuck themself often as there are harvester full of energy which wanna go to construction site near the controller and also other carrys trying to gonna fill extensions near container whitch are empty. not even sure it's the better solution now x) i'll WIP it as i'll not can work alot on next days.